build.py

```diff
@@ -10,27 +10,33 @@
 from lib.alerts import Alerts
 from lib.utils import DIST, REPO, ROOT, SRC, file_fingerprint, paragraphize
 
-jinja_loader = ChoiceLoader([
-    FileSystemLoader(str(REPO)),
-    PrefixLoader({
-        'govuk_frontend_jinja': PackageLoader('govuk_frontend_jinja')
-    })
-])
+
+def setup_jinja_environment(alerts):
```

Comment:
Minor: we could also encapsulate, and thus test, the code to decide the `target` for each kind of file. This isn't a big concern, though, as its covered by the smoke tests, with the exception of individual alert pages.